src/JSONBuilder.ts

```diff
@@ -162,7 +162,7 @@ export { __wrapper_${name} as ${name} }`);
   }
 
   static decode<V = Uint8Array>(buf: V): ${className} {
-    return decode<${className}>(buf);
+    return decode<${className}>(buf!);
```

Comment:
This results in a runtime check that it isn't null and will throw if it is.  Is this what we want?  Is the method being generic important since it will always be a Uint8Array right?

Reply:
>  Is the method being generic important 

probably no. I just did whatever to make it compile, because we need to push new version ASAP (please give me npm access).